File: dsm-platform-lib/src/main/java/transformator/ModelTransformator.java
package transformator;

import model.Model;

import java.util.UUID;

public class ModelTransformator {
    public Model createModelByDsl(Model dsl){
        return new Model("", UUID.randomUUID());
    }

    public Model transformModel(Model model, Model dslFrom, Model dslTo){
        return new Model("", UUID.randomUUID());
    }
}

File: dsm-platform-lib/src/main/kotlin/ru/psu/transformator/ModelTransformator.kt
package ru.psu.transformator

import ru.psu.model.Model
import java.util.*

class ModelTransformator {
    fun createModelByDsl(dsl: Model?): Model? {
        return Model(UUID.randomUUID(), "")
    }

    fun transformModel(model: Model?, dslFrom: Model?, dslTo: Model?): Model? {
        return Model(UUID.randomUUID(), "")
    }
}